pkg/controllers/user.go

```diff
@@ -21,10 +23,51 @@ func NewUserController(
 	return &UserController{logger, createUserInteractor}
 }
 
+func (uc *UserController) writeResponse(w http.ResponseWriter, status int, err string) {
+	js, jsonErr := json.Marshal(entities.StatusResponse{Code: status, Message: err})
```

Comment:
I think the code shouldn't be in the response body because it's already in the header.